rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -215,6 +203,58 @@ void increaseLockingCap_invalidParameter() {
         assertThrows(VMException.class, () -> bridge.execute(aboveMaxLengthData));
     }
 
+    @Test
+    void increaseLockingCap_whenNoArgumentsInTheMethodSignature_shouldReturnFalse() throws VMException {
```

Comment:
Is this right? If before it was throwing an exception and now it's returning false we have a problem

Reply:
When we run this test case expecting an `exception`, it gets an `error`. Take a look at the capture:

<img width="1394" alt="image" src="https://github.com/user-attachments/assets/7bd66779-68d0-4362-a11f-611cdedb1753">

Reply:
This is already done! When `No Arguments` Test Case, replaced `encodeSignature` with `encodeArguments` method. The name of the test is now: 
`increaseLockingCap_whenNoArgumentsInTheMethodSignature_shouldThrowVMException`